person.rb

```diff
@@ -1,36 +1,35 @@
-require_relative './corrector'
-require_relative './rental'
-
+require_relative 'corrector'
+require_relative 'rental'
 class Person
-  attr_accessor :name, :age, :rental
+  attr_accessor :name, :age, :rentals
   attr_reader :id
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     @id = Random.rand(1..1000)
-    @name = name
     @age = age
+    @name = name
     @parent_permission = parent_permission
     @corrector = Corrector.new
-    @rental = []
+    @rentals = []
   end
 
-  def validate_name
-    @name = @corrector.correct_name(name)
+  def of_age?
+    @age >= 18
   end
 
-  def add_rental(book, date)
-    Rental.new(book, date, self)
+  def can_use_services?
+    of_age? || @parent_permission
   end
 
-  private
-
-  def of_age?
-    @age >= 18
+  def validate_name
+    @name = @corrector.correct_name(@name)
   end
 
-  def can_use_services?
-    of_age? || parent_permission == true
+  def add_rental(date, book)
+    Rental.new(date, book, self)
   end
+
+  private :of_age?
 end
 
 person = Person.new(100, 'gordontinyefuza')
```

Comment:
- [ ] Don't forget to remove any remainings from the developing process like those statements  [35 - 38] 😊